### PR TITLE
docs: update consumer and upstream docs

### PR DESCRIPTION
**Pull request by hf400159**
### Description

Fixes # (issue)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/en/latest/terminology/consumer.md [hunk 1]
```diff
@@ -54,83 +62,102 @@ Consumers are useful when you have different consumers requesting the same API a
...
-...
+    ```shell
+    curl http://127.0.0.1:9080/hello -H 'apikey: auth-one' -I
+    ```
 
-# The third test returns 503 and the request is restricted
-$ curl http://127.0.0.1:9080/hello -H 'apikey: auth-one' -I
-HTTP/1.1 503 Service Temporarily Unavailable
-...
+    The third test returns 503 and the request is restricted
```
Comment (SylviaBABY) [inline docs/en/latest/terminology/consumer.md:121]:
```suggestion
    The third test returns `503` and the request is restricted.
```

File: docs/en/latest/terminology/consumer.md [hunk 2]
```diff
@@ -54,83 +62,102 @@ Consumers are useful when you have different consumers requesting the same API a
...
-        "consumer-restriction": {
-            "blacklist": [
-                "jack"
-            ]
-        }
-    },
-    "upstream": {
-        "nodes": {
-            "127.0.0.1:1980": 1
+1. Add Jack to the blacklist
```
Comment (SylviaBABY) [inline docs/en/latest/terminology/consumer.md:130]:
```suggestion
1. Add Jack to the blacklist.
```

File: docs/en/latest/terminology/consumer.md [hunk 3]
```diff
@@ -54,83 +62,102 @@ Consumers are useful when you have different consumers requesting the same API a
...
+            "nodes": {
+                "127.0.0.1:1980": 1
+            },
+            "type": "roundrobin"
+        },
+        "uri": "/hello"
+    }'
+    ```
+
+Repeated tests, all return 403; Jack is forbidden to access this API:
```
Comment (SylviaBABY) [inline docs/en/latest/terminology/consumer.md:154]:
```suggestion
2. Repeated tests, all return `403`; Jack is forbidden to access this API:
```

File: docs/zh/latest/terminology/consumer.md [hunk 4]
```diff
@@ -41,87 +56,108 @@ title: Consumer
...
+    ```
+
+    第三次测试返回 `503`，请求被限制：
+
+    ```shell
+    HTTP/1.1 503 Service Temporarily Unavailable
+    ...
+    ```
+
+通过 [consumer-restriction](../plugins/consumer-restriction.md) 插件，限制用户 `jack` 对该 Route 的访问
```
Comment (SylviaBABY) [inline docs/zh/latest/terminology/consumer.md:126]:
```suggestion
通过 [consumer-restriction](../plugins/consumer-restriction.md) 插件，限制用户 `jack` 对该 Route 的访问。
```

File: docs/zh/latest/terminology/consumer.md [hunk 5]
```diff
@@ -41,87 +56,108 @@ title: Consumer
...
+
+1. 设置黑名单，禁止 jack 访问该 API
```
Comment (SylviaBABY) [inline docs/zh/latest/terminology/consumer.md:128]:
```suggestion
1. 设置黑名单，禁止 jack 访问该 API。
```

File: docs/zh/latest/terminology/consumer.md [hunk 6]
```diff
@@ -41,87 +56,108 @@ title: Consumer
...
+            "nodes": {
+                "127.0.0.1:1980": 1
+            },
+            "type": "roundrobin"
+        },
+        "uri": "/hello"
+    }'
+    ```
+
+2. 通过以下命令访问该路由，均返回 `403`，`jack` 被禁止访问：
```
Comment (SylviaBABY) [inline docs/zh/latest/terminology/consumer.md:152]:
```suggestion
2. 通过以下命令访问该路由，均返回 `403`，`jack` 被禁止访问。
```

File: docs/zh/latest/terminology/upstream.md [hunk 7]
```diff
@@ -21,197 +27,217 @@ title: Upstream
 #
 -->
 
-Upstream 是虚拟主机抽象，对给定的多个服务节点按照配置规则进行负载均衡。Upstream 的地址信息可以直接配置到 `Route`（或 `Service`) 上，当 Upstream 有重复时，就需要用“引用”方式避免重复了。
+## 描述
 
-![Upstream 示例](../../../assets/images/upstream-example.png)
+Upstream（也称之为上游）是对虚拟主机抽象，即应用层服务或节点的抽象。你可以通过 Upstream 对象，它支持对多个服务节点按照配置规则进行负载均衡。
```
Comment (SylviaBABY) [inline docs/zh/latest/terminology/upstream.md:32]:
```suggestion
Upstream（也称之为上游）是对虚拟主机抽象，即应用层服务或节点的抽象。你可以通过 Upstream 对象对多个服务节点按照配置规则进行负载均衡。
```

**Review by SylviaBABY — DISMISSED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Comment by juzhiyuan** [comment]
Hi @hf400159, some conflicts here, please have a check.

**Review by SylviaBABY — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**